zebra-chain/Cargo.toml

```diff
@@ -41,7 +41,8 @@ proptest-derive = { version = "0.3.0", optional = true }
 displaydoc = "0.1.7"
 ed25519-zebra = "1"
 equihash = "0.1"
-redjubjub = "0.2"
+#redjubjub = "0.2"
+redjubjub = {git = "https://github.com/ZcashFoundation/redjubjub", rev = "8101eaff1cb2fca45334f77a65caa4c46e3d545b"}
```

Comment:
we should probably stick with using the `[patch.crates-io]` section in the top level toml for specifying git deps

Reply:
Let's also open a follow-up ticket that describes when we can un-patch the crates.

The existing ticket is #1086, but we should probably start a new one for crypto crates.